internals/secrethub/org_init.go

```diff
@@ -28,7 +28,9 @@ func NewOrgInitCommand(io ui.IO, newClient newClientFunc) *OrgInitCommand {
 func (cmd *OrgInitCommand) Register(r Registerer) {
 	clause := r.Command("init", "Initialize a new organization account.")
 	clause.Flag("name", "The name you would like to use for your organization. If not set, you will be asked for it.").SetValue(&cmd.name)
-	clause.Flag("descr", "A description (max 144 chars) for your organization so users will recognize it. If not set, you will be asked for it.").StringVar(&cmd.description)
+	clause.Flag("description", "A description (max 144 chars) for your organization so users will recognize it. If not set, you will be asked for it.").StringVar(&cmd.description)
+	clause.Flag("descr", "").Hidden().StringVar(&cmd.description)
+	clause.Flag("desc", "").Hidden().StringVar(&cmd.description)
```

Comment:
What happens when I use `--description` AND `--desc` at the same time? Should we do something to catch that behavior or do we say that's undefined behavior?

Reply:
The last provided value is used.

---

As there's no usecase to provide both flags and this behavior is not defined in the help nor the docs, I'd feel comfortable breaking this in the future (e.g. returning an error when both flags are supplied).

How do you feel about that?

Reply:
Yes that's cool, as long as the current behavior is consistent. So if I understand you correctly `secrethub org init --description foo --desc bar` will always use `bar` as description, correct?

Reply:
Yes, that's correct.